pkg/util/helm/helm.go

```diff
@@ -80,34 +76,52 @@ func NewHelm(param *HelmParam) (*Helm, error) {
 		CleanupOnFail:    false,
 		DryRun:           false,
 	}
-
-	helm := &Helm{
+	h := &Helm{
 		Entry:     entry,
-		ChartSpec: chartSpec,
+		ChartSpec: spec,
 		Client:    hClient,
 	}
 
-	if err = helm.AddOrUpdateChartRepo(*helm.Entry); err != nil {
```

Comment:
You separate `AddOrUpdateChartRepo()` out to `func (h *Helm) AddOrUpdateChartRepo(entry repo.Entry) error`. But without applying it.

Reply:
fixed it